src/components/pages/HomePage/TeachersButton.tsx

```diff
@@ -1,27 +1,19 @@
 /** @jsxImportSource @emotion/react */
 
-import { Button, Typography } from '@mui/material';
+import { Button } from '@mui/material';
 import { School as SchoolIcon } from '@mui/icons-material';
-import { useState, useRef } from 'react';
 
-import { getClassroom } from '@utils/classrooms';
-import BasicModal from '@components/shared/Modal';
-import ModalTextField from '@components/shared/ModalTextField';
+function getRandomPin(len: number) {
+  const min = Math.ceil(1);
+  const max = Math.floor(+''.padStart(len, '9'));
+  return Math.floor(Math.random() * (max - min) + min)
+    .toString()
+    .padStart(len, '0');
+}
```

Comment:
I think the below of adding 4 random digits is easier logic to follow.
```suggestion
function getRandomPin(pinLength: number) {
  const randomDigit = () => Math.floor(Math.random() * 10);
  const pin = [];
  for (let i = 0; i < pinLength; i++)
    pin.push(randomDigit());
  return pin.join('');
}
```

Reply:
Ah, my bad, I just updated all the items directly in the review and missed that you had those extra 2 in your comment